### PR TITLE
🏃 Swifter event deletes

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Don't delete event fires when events are deactivated
 * Don't fire event fires for inactive events

### CODE REVIEW

**Comment by nicpottier** [comment]
Take it edits are a different PR?

File: temba/campaigns/models.py [hunk 1]
```diff
@@ -508,7 +505,11 @@ def fire(self):
         Actually fires this event for the passed in contact and flow
         """
         self.fired = timezone.now()
-        self.event.flow.start([], [self.contact], restart_participants=True)
+
```
Comment (norkans7) [inline temba/campaigns/models.py]: I believe this method `fire` is not used, why not remove it?

File: temba/campaigns/views.py [hunk 2]
```diff
@@ -363,14 +364,20 @@ class CampaignEventCRUDL(SmartCRUDL):
...
+
         def get_object_org(self):
             return self.get_object().campaign.org
 
         def get_context_data(self, **kwargs):
             context = super().get_context_data(**kwargs)
             event_fires = self.get_object().event_fires.all()
 
-            fired_event_fires = event_fires.exclude(fired=None).order_by("fired", "pk")
+            fired_event_fires = event_fires.exclude(fired=None).order_by("-fired", "pk")
```
Comment (ericnewcomer) [inline temba/campaigns/views.py]: not really related, the order on the contact page for fired events should be inverted

**Comment by ericnewcomer** [comment]
@nicpottier @rowanseymour this one is ready again.

**Pull request merged**